app/assets/stylesheets/application.css.scss.erb

```diff
@@ -318,3 +318,9 @@ $service-colors: (
   color: yellow;
   text-decoration: underline;
 }
+
+.select2-results .fa {
+  float: right;
```

Comment:
It was originally floated to the left. I attached a screenshot below. let me know if I need to tinker with the css.